tests/phpunit/tests/block-supports/shadow.php

```diff
@@ -35,55 +40,71 @@ public function test_shadow_style_is_applied() {
 						'type' => 'object',
 					),
 				),
-				'supports'    => array(
-					'shadow' => true,
-				),
+				'supports'    => $supports,
 			)
 		);
-		$registry   = WP_Block_Type_Registry::get_instance();
-		$block_type = $registry->get_registered( $this->test_block_name );
-		$block_atts = array(
-			'style' => array(
-				'shadow' => '60px -16px teal',
-			),
-		);
+		$registry = WP_Block_Type_Registry::get_instance();
 
-		$actual   = wp_apply_shadow_support( $block_type, $block_atts );
-		$expected = array(
-			'style' => 'box-shadow:60px -16px teal;',
+		return $registry->get_registered( $this->test_block_name );
+	}
+
+	/**
+	 * Tests the generation of shadow block support styles.
+	 *
+	 * @dataProvider data_generate_shadow_fixtures
+	 *
+	 * @param boolean|array $support Shadow block support configuration.
+	 * @param string        $value   Shadow style value for style attribute object.
+	 * @param array         $expected       Expected shadow block support styles.
+	 */
+	public function test_gutenberg_apply_shadow_support( $support, $value, $expected ) {
+		$block_type  = self::register_shadow_block_with_support(
+			'test/shadow-block',
+			array( 'shadow' => $support )
 		);
+		$block_attrs = array( 'style' => array( 'shadow' => $value ) );
+		$actual      = gutenberg_apply_shadow_support( $block_type, $block_attrs );
```

Comment:
Thanks for bringing in the tests!

We have to rename the functions, since the `gutenberg_` prefix is meant to be used within Gutenberg and not core (https://github.com/WordPress/wordpress-develop/pull/6279/files#diff-a903d994435db8240aed96e253cfb2097ecbe9705d6f88ecdfd441422ecf2da3R52)

```suggestion
	public function test_wp_apply_shadow_support( $support, $value, $expected ) {
		$block_type  = self::register_shadow_block_with_support(
			'test/shadow-block',
			array( 'shadow' => $support )
		);
		$block_attrs = array( 'style' => array( 'shadow' => $value ) );
		$actual      = wp_apply_shadow_support( $block_type, $block_attrs );
```

Reply:
Fixed [here](https://github.com/WordPress/wordpress-develop/pull/6613/commits/5383012d66abb19d39388750fd3e1b46ec0d60b6). 

@colinduwe please pull and 
`git cherry-pick 5383012d66abb19d39388750fd3e1b46ec0d60b6`

Reply:
Done